networking/cidr-range-definitions.adoc

```diff
@@ -13,6 +13,10 @@ You must specify non-overlapping ranges for the following CIDR ranges.
 Machine CIDR ranges cannot be changed after creating your cluster.
 ====
 
+ifdef::openshift-rosa, openshift-dedicated[]
+When specifying subnet CIDR ranges, ensure that the subnet CIDR range is within the defined Machine CIDR. You must verify that the subnet CIDR ranges allow for enough IP addresses for all infrastructure and node IPs, including at least eight IP addresses for possible AWS Load Balancers.
+endif::openshift-rosa, openshift-dedicated[]
```

Comment:
I see that in our docs , we have endif::[]. Also the preview shows the conditionals , I think it should not show

Reply:
Otherwise looks good